src/ucc/api/ucc.h

```diff
@@ -14,6 +14,8 @@
 #include <ucc/api/ucc_version.h>
 #include <ucc/api/ucc_status.h>
 #include <stdio.h>
+//#include "components/mc/base/ucc_mc_base.h"
+typedef struct ucc_mc_buffer_header ucc_mc_buffer_header_t;
```

Comment:
remove. mc_buffer_header is internal implementation. no changes should be made to ucc.h